courses/utils_test.py

```diff
@@ -146,12 +170,37 @@ def test_generate_program_certificate_success(user, program):
 
     CourseRunCertificateFactory.create(user=user, course_run=course_run)
 
-    result = generate_program_certificate(user=user, program=program)
-    assert result[1] is True
-    assert isinstance(result[0], ProgramCertificate)
+    with patch.object(log, "info") as mock_log:
+        cert, created = generate_program_certificate(user=user, program=program)
+        mock_log.assert_called()
+    assert created is True
+    assert isinstance(cert, ProgramCertificate)
     assert len(ProgramCertificate.objects.all()) == 1
 
 
+def test_revoke_program_certificate(user, program):
+    """
+    Test that revoke_program_certificate revokes a program certificate
+    """
+    course = CourseFactory.create(program=program)
+    course_run = CourseRunFactory.create(course=course)
+
+    course_cert = CourseRunCertificateFactory.create(user=user, course_run=course_run)
+    assert course_cert.is_revoked is False
+
+    cert, created = generate_program_certificate(user=user, program=program)
```

Comment:
Please just create the necessary object(s) directly instead of running `generate_program_certificate` to create them, and remove any testing assertions that are repeated from the test above (like `assert created is True`)